server/chat-commands.js

```diff
@@ -3740,6 +3740,40 @@ const commands = {
 	},
 	importinputloghelp: [`/importinputlog [inputlog] - Starts a battle with a given inputlog. Requires: + % @ & ~`],
 
+	acceptdraw: 'acceptdraw',
+	offerdraw(target, room, user, connection, cmd) {
+		const battle = room.battle;
+		if (!battle) return this.errorReply("Must be in a battle room.");
+		if (!Config.allowrequestingties) return this.errorReply("This server does not allow the offering/accpeting of draws");
+		if (!this.can('roomvoice', null, room)) return;
+		if (cmd === 'acceptdraw' && !battle.allowTie) return this.errorReply("No draw offer available. It might have been automatically withdrawn as the current turn started.");
```

Comment:
Better phrasing, also the property on `Player`

```suggestion
		if (cmd === 'acceptdraw' && !battle.players.some(player => player.wantsTie)) {
			return this.errorReply("No other player is requesting a tie right now. It was probably canceled.");
		}
```